translations/de.ts

```diff
@@ -673,15 +627,10 @@ History will be disabled!</source>
 Die Verlaufsfunktion wird erst einmal ausgeschaltet!</translation>
     </message>
     <message>
-        <source>Please enter the password for the chat history for the %1 profile.</source>
-        <comment>used in load() when no hist pw set</comment>
-        <translation type="vanished">%1, bitte gib dein Gesprächsverlaufpasswort ein.</translation>
-    </message>
-    <message>
         <location filename="../src/core/coreencryption.cpp" line="127"/>
         <source>Please enter the password for the chat history for the profile &quot;%1&quot;.</source>
         <comment>used in load() when no hist pw set</comment>
-        <translation>%1, bitte gib dein Chatverlaufpasswort ein.</translation>
+        <translation type="unfinished">%1, bitte gib dein Chatverlaufpasswort ein.</translation>
```

Comment:
Yes, because I wanted that another person will also look at this sentence. Doesn't look great.